db/migrate/20171128094007_add_omniauth_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddOmniauthToUsers < ActiveRecord::Migration[5.1]
```

Comment:
you need to create a new table for omniauth providers for future multiple providers